zboss/README.rst

```diff
@@ -5,20 +5,14 @@ ZBOSS Zigbee stack
 
 The |NCS|'s :ref:`nrf:ug_zigbee` stack uses ZBOSS – a portable, high-performance Zigbee software protocol stack that allows for interoperability, customizing, testing, and optimizing of your Zigbee solution.
 
-The nrfxlib repository contains the following versions of the ZBOSS libraries:
+The nrfxlib repository contains the following version of the ZBOSS libraries:
 
 * *Production* version that contains the latest stable ZBOSS libraries.
```

Comment:
I don't recall there being that many ZBOSS updates lately, so is saying that they contain the latest stable libraries correct?

Reply:
The intention is to update the libraries in this same PR.